src/external/modules/view-licences/controller.js

```diff
@@ -100,11 +101,12 @@ async function postLicenceRename (request, h) {
 
   // Rename licence
   const { name } = getValues(form);
-  const { error } = await services.crm.documents.setLicenceName(documentId, name);
+  const rename = !!documentName;
```

Comment:
The rename param would not be needed here if this is changed in the water service

Reply:
change applied